src/modules/item/export.ts

```diff
@@ -118,6 +126,33 @@ function buildItemEntityMetadata(collection: Collection, item: Item): CatalystIt
   }
 }
 
+function buildTPItemEntityMetadata(item: Item, itemHash: string, tree: MerkleDistributorInfo): TPCatalystItem {
+  if (!item.urn) {
+    throw new Error('Item does not have URN')
+  }
+  const baseEntityData = {
+    id: item.urn,
+    ...getBaseEntityMetadata(item)
+  }
+  return {
+    merkleProof: getMerkleProof(tree, itemHash, baseEntityData),
+    ...baseEntityData
```

Comment:
The TP item will need the `content` property, where its keys are file names and its values are the hashes of the files.

Reply:
updated! thanks for catching that it was missing :D